dist/expand-button/ds6/expand-button.css

```diff
@@ -312,6 +312,12 @@ button.expand-btn--large,
 button.expand-btn--extra-large {
   font-size: 1.125rem;
 }
+button.expand-btn--small {
+  padding: 7px 16px;
+}
+button.expand-btn--small.expand-btn--no-text {
```

Comment:
I think this gets the job done for now, and fixes the issue. But modifier chaining in BEM is a code smell, as modifiers are supposed to be completely independent of each other. So at some point in the future we can rethink whether perhaps we need a new base class or not, or whether one of these modifiers could go on a child class instead of the base.

Reply:
Sure. Trying to be least-disruptive at the moment.